homeassistant/components/switcher_kis/__init__.py

```diff
@@ -63,7 +63,7 @@
 
     try:
         device_data = await wait_for(
-            v2bridge.queue.get(), timeout=5.0, loop=hass.loop)
+            v2bridge.queue.get(), timeout=10.0, loop=hass.loop)
```

Comment:
This highlighted that your test was actually waiting the full timeout. That is not good.Stub it out.

Reply:
Yeah sorry about that. :-)

Resolved by adding a "failed bridge" fixture to mock failure without waiting the full time out [here](https://github.com/home-assistant/home-assistant/pull/23930/commits/6247a6d2976ceb8443cddbd230bf8586cfb4dca5).